### ISSUE TITLE
IDS 1.0 schema accepts multiple entity facets in the requirements: is it valid?

### ISSUE THREAD

**Issue by giuseppeverduciALMA** [buildingSMART/IDS#338]
The IDS 1.0 schema allows multiple entities in the requirements section.
An attached example demonstrates this, which I believe is unintended behavior.
[entities.zip](https://github.com/user-attachments/files/16749487/entities.zip)

**Comment by atomczak** [comment]
yes, I agree this is undesired. #344 provides a possible solution.

**Comment by BIMvoice** [comment]
Ran your `entities.ids` through IfcOpenShell's IfcTester against a model containing `IfcWall` in `PARAPET`, `MOVABLE` and `NOTDEFINED`.

**Direct answer: yes, multiple entity facets in requirements are currently valid IDS.** It is the schema as written rather than a validator being lenient. `requirementsType`'s outer sequence carries `maxOccurs="unbounded"` and the `entity` element has no cap of its own:

```xml
<xs:complexType name="requirementsType">
    <xs:sequence maxOccurs="unbounded">
        <xs:element name="entity" minOccurs="0">
```

Identical on `development` and `ver/1.1.x`.

The contrast is the interesting part. `applicabilityType` uses a plain sequence with no `maxOccurs`, so a second `entity` facet there is structurally rejected. I confirmed both sides: your file loads fine, while the same pair of facets moved into applicability fails schema validation and IfcTester refuses to load it.

**What your file actually does.** The two facets are ANDed, so a wall would have to be `PARAPET` and `MOVABLE` at once. No wall can satisfy that, and it fails for all three in my model. It does not silently pass. I also tried two facets naming different classes, and two identical facets, looking for a case that wrongly reports compliance, and could not produce one. So the practical damage is an unsatisfiable specification rather than a false pass.

This is the same defect as #344, where the schema side is being discussed. Current status there, since it is not obvious from this thread: you agreed on 2024-11-06 to drop `maxOccurs="unbounded"` to prevent exactly this, then a week later proposed `<xs:choice maxOccurs="unbounded">` instead. Neither has a PR on either branch. The two issues cross-reference each other in their timelines but not in their discussions, which is probably why the status is hard to see from here.
